app/Services/AdventuresLeagueAdapter.php

```diff
@@ -83,33 +86,77 @@ public function getCharacter($filePath): ?Character
     private function getEntries($data, $characterId)
     {
         $entries = [];
+
+        // entries only begin on line 5 (index 4) of the exported csv
         for ($i = 4; $i < count($data); $i++) {
-            //Check if row is a magic item entry
             $isItemEntry = ($data[$i][0] == "MAGIC ITEM");
+            $isCharacterLogEntry = ($data[$i][0] == "CharacterLogEntry");
+            $isCampaignLogEntry = ($data[$i][0] == "CampaignLogEntry");
+            $isDmLogEntry = ($data[$i][0] == "DmLogEntry");
+            $isTradeLogEntry = ($data[$i][0] == "TradeLogEntry");
+            $isPurchaseLogEntry = ($data[$i][0] == "PurchaseLogEntry");
+
+            $isEntryLine = ($isCharacterLogEntry || $isDmLogEntry || $isCampaignLogEntry || $isTradeLogEntry || $isPurchaseLogEntry);
```

Comment:
This could probably be extracted to a function? What do you think? or maybe it can just become
```php
$isEntryLine = in_array($data[i][0], ["CharacterLogEntry", "CampaignLogEntry", "DmLogEntry", "TradeLogEntry",  "PurchaseLogEntry"]
```

Additionally, we might need to reconsider if we will be turning `TradeLogEntry`s into entries in our system or not, or if we'll create dynamic entries from trades; currently i'm undecided :)

Reply:
Im using those individual bools in other combinations elsewhere though so i think keeping them atomic is best.